substrate/runtime-io/without_std.rs

```diff
@@ -144,6 +167,21 @@ pub fn storage_root() -> [u8; 32] {
 	result
 }
 
+/// The current storage' changes root.
+pub fn storage_changes_root() -> Option<[u8; 32]> {
+	let mut is_set: u8 = 0;
+	let mut result: [u8; 32] = Default::default();
+	unsafe {
+		ext_storage_changes_root(&mut is_set, result.as_mut_ptr());
```

Comment:
Why we need `is_set` here? Wouldn't it be easier to just return it?

Reply:
`Option` is not `ConvertibleToWasm`. It is easier to return `Option` than `is_set` + `result`.

Reply:
sorry should have state it more clearer: what i meant is that we can return is_set not via pointer but by the return value

Reply:
Ok, thanks! :)